learn.md

```diff
@@ -55,13 +54,13 @@ permalink: /learn/
    </div>
    <div class="col-sm-12 col-md-4 cBoxContainer">
       <div class="cBallerina-Box">
-     <p class="cBoxpara">Learn the “batteries-included” standard library</p>
+     <p class="cBoxpara">Learn the “batteries-included” standard library.</p>
          <a class="cBallerinaLearnButtons" href="/learn/api-docs/ballerina">The Standard Library</a>
       </div>
    </div>
    <div class="col-sm-12 col-md-4 cBoxContainer">
       <div class="cBallerina-Box">
-     <p class="cBoxpara">Read the language & platform specifications to really understand the guts of the language.</p>
+     <p class="cBoxpara">Read the language & platform specifications to get an in-depth understanding of the language and its design.</p>
```

Comment:
Not related to this PR, but wondering if we should remove "platform specifications" for the time-being, since we don't have one to list? @anupama-pathirage thoughs?

Reply:
The content of the current spec is organized in a manner to consist specifications of both the language and platform. Therefore, we should be able to keep this as it is.

<img width="1224" alt="Screenshot 2019-11-21 at 10 08 59" src="https://user-images.githubusercontent.com/11707273/69304571-199aad00-0c47-11ea-8592-3e215f8b9de5.png">

Reply:
What I meant was there is no "platform specifications" to read?

Reply:
Yeah, as discussed with @anupama-pathirage and @hasithaa , we can keep this as it is since the platform specifications will be added very soon.

Reply:
@praneesha It seems with this change we have introduced a alignement issue in the website. Can u pls look into this?

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/16300038/69853107-f9ed2f80-12ab-11ea-9ab1-336e9863d5b9.png">

Reply:
@madhuramendis - Can you help to fix this, please?

Reply:
IMO, The sentence is too long, Can we consider having a shorter but catchy sentence as in the previous two boxes?

Reply:
We can say "Get an in-depth understanding of the language and its design". I feel the "Read the language & platform specifications" part is redundant.

Reply:
+1

Reply:
Fixed this in https://github.com/ballerina-platform/ballerina-dev-website/pull/126/files.